pyproject.toml

```diff
@@ -0,0 +1,92 @@
+[build-system]
+requires = ["setuptools>=61.0.0", "wheel"]
+build-backend = "setuptools.build_meta"
+
+[project]
+name = "open-flash"
+version = "0.1.9"
+authors = [
+  {name = "SEA LAB"} 
+]
+description = "A Python package for semi-analytical hydrodynamics modeling including matched eigenfunction expansion method"
+readme = "README.md"
+requires-python = ">=3.8"
+classifiers = [
+    "Programming Language :: Python :: 3",
+    "Programming Language :: Python :: 3.8",
+    "Programming Language :: Python :: 3.9",
+    "Programming Language :: Python :: 3.10",
+    "Programming Language :: Python :: 3.11",
+    "Programming Language :: Python :: 3.12",
+    "Operating System :: OS Independent",
+    "Intended Audience :: Science/Research",
+    "Topic :: Scientific/Engineering :: Physics",
+]
+dependencies = [
```

Comment:
instead of super specific versions and all packages from an env export, this should be a minimal list of dependencies with inequalities